app/services/ipam/eui64.rb

```diff
@@ -24,7 +24,7 @@ def validate
 
     def mac_to_ip(mac)
       # cleanup MAC address
-      mac.gsub!(/[\.\:\-]/, '')
+      mac.gsub!(/[.:\-]/, '')
```

Comment:
Why not:
```suggestion
      mac.gsub!(/[.:-]/, '')
```

Reply:
> However, I think changing `.` to `\.` is a significant change that will likely break things.

Some limited testing suggests that you can't escape `.`, but IMHO needs some further investigation. Ideally also check test coverage for these methods.